packages/larva-patterns/components/c-label/c-label.prototype.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	c_label_text: 'Label Here',
+	c_label_tag: "span",
```

Comment:
Single quotes in these object values - we don't have a linting / autofix config setup here unfortunately.